// Echo 服务的 Protobuf 定义
syntax = "proto3";

package example;
option cc_generic_services = true;

// 请求消息
message EchoRequest {
    string message = 1;
}

// 响应消息
message EchoResponse {
    string message = 1;
}

// Echo 服务
service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}
